apps/balancer-tools/src/components/SearchPoolForm/index.tsx

```diff
@@ -191,8 +192,10 @@ export function SearchPoolForm({
                         <span>
                           {/* By default, FX pools have the name equal to "BPT".
                             So, we'll use name instead of the symbol, since it is more meaningful */}
-                          {poolTypeFilter?.[0] == "FX" &&
-                          poolTypeFilter?.length == 1
+                          {gqlAdditionalVariable?.poolTypeFilter &&
+                          Array.isArray(gqlAdditionalVariable.poolTypeFilter) &&
+                          gqlAdditionalVariable.poolTypeFilter?.[0] == "FX" &&
+                          gqlAdditionalVariable.poolTypeFilter?.length == 1
```

Comment:
Something here broke the name edge case for FX

Production:
<img width="517" alt="image" src="https://github.com/bleu-fi/balancer-tools/assets/63298341/b46740a1-bdb4-4e3b-bc54-9004f2b27271">

This branch preview

<img width="517" alt="image" src="https://github.com/bleu-fi/balancer-tools/assets/63298341/fb2f89fc-7888-4b59-8477-b63c19d022db">